### DISCUSSION TITLE
Remove Failed Device

### DISCUSSION THREAD

**Discussion by vipul128** [tsightler/ring-mqtt#626] [Support Q&A]
I had to remove a failed device from my Ring system and ended up replacing it with the same sensor when I added it back in.  How do I remove the failed device from HA?  I see the replaced device got added as a new device and entities.  I was able to delete the old entities but cannot delete the old device.

**Comment by tsightler** [comment]
Why are you not able to delete the device?  Note that you must restart ring-mqtt for it to forget about previous devices, but you should be able to delete devices from HA at any time.
